tests/conftest.py

```diff
@@ -87,6 +92,46 @@ def values_str() -> TListValuesInputStr:
     return [uuid_str(), uuid_str(), uuid_str()]
 
 
+@pytest.fixture
+def dictionary_name() -> TDictionaryName:
+    return uuid_str()
+
+
+@pytest.fixture
+def dictionary_field() -> TDictionaryField:
+    return uuid_str()
+
+
+@pytest.fixture
+def dictionary_value() -> TDictionaryValue:
+    return uuid_str()
+
+
+@pytest.fixture
+def dictionary_field_bytes() -> TDictionaryField:
+    return uuid_bytes()
+
+
+@pytest.fixture
+def dictionary_value_bytes() -> TDictionaryValue:
+    return uuid_bytes()
+
+
+@pytest.fixture
+def dictionary_fields() -> TDictionaryFields:
+    return [uuid_str(), uuid_str(), uuid_bytes(), uuid_bytes()]
+
+
+@pytest.fixture
+def dictionary_items() -> TDictionaryItems:
+    return dict([(uuid_str(), uuid_str()), (uuid_bytes(), uuid_bytes())])
+
+
+@pytest.fixture
+def increment_amount() -> int:
+    return random.randint(0, 42)
```

Comment:
We can increment a negative number, right?

Reply:
Yes that's right. Are you saying we should add negative numbers to the range?

Reply:
Yes, that will give us full coverage in case someone validates it wrong.

Reply:
Fixed in c200d51